app/Http/Resources/CastResource.php

```diff
@@ -17,7 +18,7 @@ public function toArray($request)
         return [
             'id' => $this->id,
             'medium_id' => $this->medium_id,
-            'person_id' => $this->person_id,
+            'person_data' => new PersonResource(Person::findOrFail($this->id)),
             'gender_id' => $this->gender_id,
             'character' => $this->character,
             'created_at' => $this->created_at->format('d-m-Y'),
```

Comment:
format ok but look for how to add the month in words

Reply:
We decided to use number and then on the frontend we gonna transfer it to month name

Reply:
Ok then no problem :D